python/scripts/cirkopt.py

```diff
@@ -286,6 +270,10 @@ def search(self):
             datefmt="%I:%M:%S %p",
             level=args.loglevel,
         )
+
+        if args.seed is None:
+            args.seed = randint(0, 2 ** 32)  # default to random seed, but print it
```

Comment:
technically this could be done via the default argument right?

```
        parser.add_argument(
            "--seed",
            help="Removes randomization on the initial seed to make the results more reproducable",
            type=int,
            default=randint(0, 2 ** 32),
        )
```

Reply:
I think I was originally going to do it differently